acceptance/tests/resource/user/should_modify_home.rb

```diff
@@ -1,6 +1,6 @@
 test_name "should modify the home directory of an user on OS X < 10.14" do
   confine :to, :platform => /osx/
-  confine :except, :platform => /osx-10.1[4-9]/
+  confine :except, :platform => /(osx-10.1[4-9]|osx-11-)/
```

Comment:
We get now in a weird spot because this tests are not run on any of the current supported platforms, but is also good to keep to to see what is not working in the future mac OS versions, or to investigate if we can fix this functionality.